specs/0117-email-opt-in-unsubscribe.md

```diff
@@ -0,0 +1,97 @@
+
+This page also should not be shown if the user has opted-in via their settings.
+
+(Section: content)
+- Header: "Update your email communication preferences"
+- Sub-header: "You can now choose your email communication preferences within Pontoon."
+- Body: "Would you like to receive updates related to localization at Mozilla, projects on Pontoon, and updates to the Pontoon platform by email?”
+- Consent: “By enabling email updates, I’m okay with Mozilla handling my personal information as explained in this <a href="https://www.mozilla.org/en-US/privacy/websites/">Privacy Notice</a>."
+- Button: "Enable email updates”
+- Button: "No, thank you"
+- Body: “You can update your email communication preferences at any time in your settings."
+
+## New account creation email opt-in
+
+Ideally, we would like users to set up their email communication preferences right away upon successful account creation.
+
+After a user successfully creates their Pontoon account, display a standalone page which redirects to the homepage after the user enables or rejects email updates. This page informs the user of successful account creation and request for updating email communication preferences.
+
+They should be shown two buttons, one to opt-in to emails or one declining to opt-in. This should not be shown again if the user clicks either of the buttons, but the standalone page for email consent (above) should be shown again upon next login if the user closed their tab/browser.
+
+(Section: content)
+- Header: "Welcome to Pontoon!"
+- Sub-header: "You have successfully created your Pontoon account."
+- Body: "Would you like to receive updates related to localization at Mozilla, projects on Pontoon, and updates to the Pontoon platform? This can be changed at any time in your settings."
+- Button: "Enable email updates”
+- Button: "No, thank you"
+- Consent: “By enabling email updates, I’m okay with Mozilla handling my personal information as explained in this <a href="https://www.mozilla.org/en-US/privacy/websites/">Privacy Notice</a>."
+
+## Additional card for tour
+
+As part of new user onboarding, point new users to user preferences and call out the ability to enable email communication preferences.
+
+(Section: content)
+- Header: Stay up to date
+- Body: "You can receive updates related to localization at Mozilla, projects on Pontoon, and updates to the Pontoon platform by email. If you'd like to stay up to date, you can enable these emails in your settings."
+
+*Display if user logged in:*
+- Button: "Enable email updates"
+- Consent: “By enabling email updates, I’m okay with Mozilla handling my personal information as explained in this <a href="https://www.mozilla.org/en-US/privacy/websites/">Privacy Notice</a>."
+
+## Unsubscribe page
+
+Legal regulations (e.g. CAN-SPAM act) have specific requirements to include an unsubscribe link in commercial email messages to manage email preferences without requiring a log-in.
+
+As a user I can:
```

Comment:
This section is different from the rest of the specs. 

I think we need to call out the various part of the page, including what text and options we're going to display.